prow/cmd/deck/static/plugin-help-script.js

```diff
@@ -361,3 +420,17 @@ function redraw() {
         });
     redrawHelpTable(repoSel, pluginsWithCommands);
 }
+
+
+/**
+ * Extracts a command name from a command example.
+ * @param {string} commandExample
+ * @return {string}
+ */
+function extractCommandName(commandExample) {
+    const command = commandExample.split(" ");
+    if (!command || command.length === 0) {
+        throw new Error("Cannot extract command name.");
+    }
+    return command[0].slice(1).split("-").join("_");
```

Comment:
*I* know what this does, but maybe document what this is actually doing? It's not going to be immediately obvious why this does slice etc.

Reply:
done